src/main/java/org/jabref/pdfimport/PdfImporter.java

```diff
@@ -224,8 +224,10 @@ private void doContentImport(String fileName, List<BibEntry> res) {
 
     private Optional<BibEntry> createNewEntry() {
         // Find out what type is desired
-        EntryTypeDialog etd = new EntryTypeDialog(frame);
-        etd.setVisible(true);
+        //EntryTypeDialog etd = new EntryTypeDialog(frame);
+        EntryTypeView etd = new EntryTypeView(frame.getCurrentBasePanel(), frame.getDialogService(), Globals.prefs);
+        //etd.setVisible(true);
+        etd.showAndWait();
```

Comment:
The code below looks like yet another place with code duplication (which is however slightly different to the one above, eg. timestamp generation).